include/wx/private/webrequest.h

```diff
@@ -187,6 +187,8 @@ class wxWebResponseImpl : public wxRefCounterMT
 
     void ReportDataReceived(size_t sizeReceived);
 
+    void SetBufferSize(size_t);
```

Comment:
Very minor, but looking at this function name it's not really clear why/when would you call it. I think it would be nice to either rename it or write a short comment explaining it. Or both, of course:
```suggestion
    // This function can optionally be called to preallocate the read buffer,
    // if the total amount of data to be downloaded is known in advance.
    void PreAllocBuffer(size_t sizeNeeded);
```

Reply:
done in 48cba4f